### ISSUE TITLE
[2.2.0]Bug or feature? Relative path route to root.

### ISSUE THREAD

**Issue by kamicloud** [saleem-hadad/larecipe#161]
https://github.com/saleem-hadad/larecipe/commit/9ef07bbca51cddd18c42a20e3fd07cef5bfbd98c#diff-a917fe07e275932a280a02bec949accc

**Describe the bug**
When I use relative path in 2.2.0
````
  - [AccessTokenPlatform](#AccessTokenPlatform)
  - [NotificationEventStatus](#NotificationEventStatus)
````

The link in browser route to http://localhost/#AccessTokenPlatform

**To Reproduce**
1. Create a clean laravel project
2. Require larecipe 2.2.0
3. Add above doc

**Expected behavior**

Move to the correct position in current page.

I think the problem was added in [this commit](https://github.com/saleem-hadad/larecipe/commit/9ef07bbca51cddd18c42a20e3fd07cef5bfbd98c#diff-a917fe07e275932a280a02bec949accc), but I can't find the issue, why it was added

The base should be 
<base href="{{ Config::get('app.url') }}" />
or removed directly.

Is it a bug or feature? And how can I fix it in 2.2.0

**Comment by RichardDominik** [comment]
After upgrade from 1.x to 2.x I have the same problem.

**Comment by WillGoldstein** [comment]
@RichardDominik Is this issue related to getting re-routed to the /login page when `larecipe.settings.auth` is `true` and you're logged in?

**Comment by WillGoldstein** [comment]
Me too. I don't understand why this
https://github.com/saleem-hadad/larecipe/commit/9ef07bbca51cddd18c42a20e3fd07cef5bfbd98c#diff-a917fe07e275932a280a02bec949accc

was made. Deleting that addition fixes the issue.

**Comment by AbbyJanke** [comment]
I submitted a PR to fix this, based on @WillGoldstein's comment. I tested it to make sure nothing else was effected.

**Comment by WillGoldstein** [comment]
thanks @AbbyJanke

**Comment by WillGoldstein** [comment]
This was fixed with #167 and can be blosed

**Comment by jadamec** [comment]
I've same issue now.

**Comment by WillGoldstein** [comment]
@jadamec the issue is fixed but see #171 for why you don't have the fix. Just delete the one line as seen in #167 for now.
